scipy/optimize/tests/test_cython_optimize.py

```diff
@@ -0,0 +1,90 @@
+"""
+Test Cython optimize zeros API functions: ``bisect``, ``ridder``, ``brenth``,
+and ``brentq``, in `scipy.optimize.cython_optimize`, by finding the roots of a
+3rd order polynomial given a sequence of constant terms, ``a0``, and fixed 1st,
+2nd, and 3rd order terms in ``args``.
+
+.. math::
+
+    f(x, a0, args) =  ((args[2]*x + args[1])*x + args[0])*x + a0
+
+The 3rd order polynomial function is written in Cython and called in a Python
+wrapper named after the zero function. See the private ``_zeros`` Cython module
+in `scipy.optimize.cython_optimze` for more information. 
+"""
+
+from __future__ import division, print_function, absolute_import
+import numpy.testing as npt
+from scipy.optimize.cython_optimize import _zeros
+
+# constants
```

Comment:
I believe this example is solving `x**3=a` for `a=2, 2.1, 2.2, ... 2.9`.  It's a good example, simple enough but not too simple. (I.e. it's not a quadratic)
State this explicitly right after the `#constants`, as you want to make it clear to any new reader or maintainer what this is attempting to do.
E.g.
```
Solve x**3 - a = 0  for a=2.0, 2.1, ... 2.9.
The ARGS have 3 elements just to show how this could be done for any cubic polynomial.
```